### DISCUSSION TITLE
Saludos y apoyo

### DISCUSSION THREAD

**Discussion by rramosp** [rramosp/ai4eng.v1#31] [00 - General]
Hola a tod@s, 

gracias a todo el mundo por participar en la aventura de este curso y por las preguntas y sugerencias que nos estáis mandando en este foro.

A partir de ahora Felipe Rodriguez  [Felipe-RA@github](https://github.com/Felipe-RA) nos va a ayudar a dar respuesta a las preguntas que vayan surgiendo. Felipe es monitor del curso análogo que tenemos en la Universidad de Antioquia y lo conoce muy bien pues ha estado acompañándome desde la creación del mismo.

Somos ya una comunidad de más de 1000 personas inscritas en el curso, principalmente desde Colombia, pero también con una presencia significativa de toda América Latina.

Gracias a todos.
Saludos
Raúl

**Comment by ljaimesm** [comment]
Muchas gracias, espero lograr el 90% aunque hay tareas que no he podido hacer.
